runtime/registry.go

```diff
@@ -110,6 +122,11 @@ func (r *Runtime) DeleteInstance(ctx context.Context, instanceID string, dropOLA
 		if err != nil {
 			r.logger.Error("could not drop database", zap.Error(err), zap.String("instance_id", instanceID), observability.ZapCtx(ctx))
 		}
+
+		// We drop all data if dropOLAP is true
+		if err := os.RemoveAll(filepath.Join(r.opts.DataDir, instanceID)); err != nil {
+			r.logger.Error("could not drop instance data directory", zap.Error(err), zap.String("instance_id", instanceID), observability.ZapCtx(ctx))
+		}
```

Comment:
`dropOLAP` was added just for DuckDB, we never use it (and probably shouldn't use it) for external OLAPs.

So how about just always deleting the data dir and removing the `dropOLAP` parameter and `drivers.Drop` function altogether?